pandas/core/groupby.py

```diff
@@ -3232,8 +3232,9 @@ def _cython_agg_blocks(self, how, alt=None, numeric_only=True):
             locs = block.mgr_locs.as_array
             try:
                 result, _ = self.grouper.aggregate(
-                    block.values, how, axis=agg_axis)
+                    block.values, how, skipna=skipna, axis=agg_axis)
             except NotImplementedError:
+                continue
```

Comment:
take this out. This is a fairly complicated workflow of falling back on non-numerics. This is the key here.